patches/minecraft/net/minecraft/client/renderer/block/model/BlockPart.java.patch

```diff
@@ -0,0 +1,20 @@
+--- ../src-base/minecraft/net/minecraft/client/renderer/block/model/BlockPart.java
++++ ../src-work/minecraft/net/minecraft/client/renderer/block/model/BlockPart.java
+@@ -110,16 +110,7 @@
+ 
+             private float func_178255_b(JsonObject p_178255_1_)
+             {
+-                float f = JsonUtils.func_151217_k(p_178255_1_, "angle");
+-
```

Comment:
Instead of deleting all this code, just insert one line here:
```java
if (true) return f; // Forge: Remove limits on JSON BlockPart rotation angles
```